peps/pep-0757.rst

```diff
@@ -90,8 +89,11 @@ Export API
 Export a Python integer as a digits array::
 
     typedef struct PyLong_DigitArray {
```

Comment:
I suggest to rename the structure to `PyLongExport`.

Reply:
This clash with the function name.

Reply:
The function has an underscore :-)

Reply:
See also my implementation, updated to use `PyLongExport` structure (type): https://github.com/python/cpython/pull/121339

Reply:
I see)  That's why I'm not sure.  Seems too close.

Reply:
In Python, functions usually take an underscore in their name after the domain: `PyLong_XXX`, and types have no underscore after the domain: `PyLongXXX`. Example: `PyTuple_New()` function and `PyTupleObject` type name.

Either we keep "digits array" for the function name and the type name, or we move to "export" for both, for consistency.

I'm not comfortable with using "export" for the function and "digits array" for the type name. You renamed the function to "export", I suppose that it's because there is a case when you don't get an array, but "a value". In this case, I would prefer to rename the type to "export" as well.